etc/.jshintrc

```diff
@@ -0,0 +1,13 @@
+{
+  "bitwise": true,
+  "curly": true,
+  "eqeqeq": true,
+  "freeze": true,
+  "futurehostile": true,
+  "latedef": "nofunc",
+  "noarg": true,
+  "nonew": true,
+  "shadow": false,
+  "strict": false,
+  "undef": false
```

Comment:
Is it possible to use `extends` here and below? https://github.com/jshint/jshint/releases/tag/2.5.1